### ISSUE TITLE
Jenkins marian-dev-cuda-10.2 #78 failed

### ISSUE THREAD

**Issue by marianminion** [marian-nmt/marian-dev#854]
Build 'marian-dev-cuda-10.2' is failing!

Last 50 lines of build output:

```
Started by user Roman Grundkiewicz
Running as SYSTEM
Building on master in workspace /var/lib/jenkins/workspace/marian-dev-cuda-10.2
The recommended git tool is: NONE
No credentials specified
 > git rev-parse --resolve-git-dir /var/lib/jenkins/workspace/marian-dev-cuda-10.2/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/marian-nmt/marian-dev # timeout=10
Fetching upstream changes from https://github.com/marian-nmt/marian-dev
 > git --version # timeout=10
 > git --version # 'git version 2.7.4'
 > git fetch --tags --progress https://github.com/marian-nmt/marian-dev +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
Checking out Revision bb6092da2b89c3882aa256480cef13081fd6f50f (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f bb6092da2b89c3882aa256480cef13081fd6f50f # timeout=10
Commit message: "Compute tensor size using integers (#851)"
 > git rev-list --no-walk bb92b817dd5140523af9e4d756a34c2092d03e32 # timeout=10
[marian-dev-cuda-10.2] $ /bin/sh -xe /tmp/jenkins8933209325177980224.sh
+ . /etc/environment
+ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
+ rm -rf build
+ mkdir -p build
+ cd build
+ cat /var/lib/jenkins/cuda-10.2/version.txt
CUDA Version 10.2.89
+ CC=/usr/bin/gcc-8 CXX=/usr/bin/g++-8 CUDAHOSTCXX=/usr/bin/g++-8 cmake -DCOMPILE_TESTS=ON -DCOMPILE_EXAMPLES=ON -DCOMPILE_KEPLER=ON -DCOMPILE_MAXWELL=ON -DCUDA_TOOLKIT_ROOT_DIR=/var/lib/jenkins/cuda-10.2 -DUSE_SENTENCEPIECE=ON -DCOMPILE_SERVER=ON ..
CMake Error at /usr/share/cmake-3.5/Modules/CMakeDetermineCXXCompiler.cmake:56 (message):
  Could not find compiler set in environment variable CXX:

  /usr/bin/g++-8.

Call Stack (most recent call first):
  CMakeLists.txt:8 (project)


CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
See also "/var/lib/jenkins/workspace/marian-dev-cuda-10.2/build/CMakeFiles/CMakeOutput.log".
Build step 'Execute shell' marked build as failure
Skipped archiving because build is not successful

```

Changes since last successful build:
- [hihoan] 96ed0baf5a8a0c39508fb53ff13e0ab309fd7cea - chmod -x

- [hihoan] 0d8372c590b290dc9b10200ea068c407954229a8 - move embedding to its own file

- [hihoan] ca47eabca5cb9bb11a3e4fe45afa77501128b4b9 - move output to its own file

- [hihoan] f7266886f0d478d802a88f2ce82b71f27c37bf07 - move logits to its own file

- [hihoan] 42406cc715a301db2c3087d0def6ff70332d1074 - move logits to its own file

- [hihoan] b88c3fcb71ad06326f60cc930fd283d77cd41a6c - costs.cpp

- [hihoan] 085c8a7a98a462ea809dc4efcff7ba0f5b49066d - more code from .h -> .cpp

- [hihoan] 7c1cb8462a2adc6540ee78f111a75ed4fbdd66ad - add logits.cpp

- [hihoan] 55f4216552bca148091f15b72c5c2e5b486d4c79 - add .h

- [hihoan] ba196637847c50c76d5d0edfcfe39b9cedb0d1d0 - clang-format -i

- [Roman Grundkiewicz] cd018e8d0404687c0bd13f64962bd22617b80331 - Update formatting

- [github] 77c3e356a47113f661dda794b815e84561ca93f5 - Install Boost in GitHub workflows (#834)

- [Roman Grundkiewicz] a1aaa32c6af15f9bc653b6ceeb81ac1767ef3f39 - Merged PR 18201: Install Boost in Azure pipelines

- [Marcin.JunczysDowmunt] e08c52a8df4be5cf2d21cbb6e1eb3a5dcff0d0e2 - Merged PR 18185: Support for Microsoft legacy binary shortlist

- [Marcin Junczys-Dowmunt] 8f73923d3134f4799497b7e880963336b8fe4d6b - increase version and update changelog

- [github] 571634a2c05b9bc2de206b1061fbc39a6e97087a - Update GitHub workflows with Ubuntu+CUDA (#837)

- [Roman Grundkiewicz] 326b9400e9b4dfa6bbf445e4cbbc83ea5145b6b7 - Merged PR 18232: Update VS CMake builds and scripts

- [github] db2a5e4d66e92dbed7711ace8393f3fd618b89f4 - Fix broken links to MNIST data sets (#838)

- [github] a11418c17c5e295ddf5bfd7b9963cfec78e3852a - Add simple unit tests for binary files (#826)

- [github] c724837ab326ceab629e30f7a7ae89e45065f3df - Update VERSION

- [github] d780082973f7de03c9426a5b2fa7936408282114 - Fix model loading on architectures where size_t is 32bits (#825)

- [github] c89efbe9192974664ef8a54b99f2202c0ac12411 - Update VERSION

- [github] 096c48e51cd2e61bb275345d7cca99cbfd6bc5c7 - remove TC_MALLOC from optional dependencies (#840)

- [github] 0394d2cdbe42b7ad7fc02854da8e063f2a23496f - Display decoder speed statistics with --stat-freq N (#841)

- [github] b36d0bbbab1fa4d435e517c974c3fde96e3145fe - Fix FBGEMM build with gcc 9.3+ (#836)

- [hihoan] 415769fb2f953c5abf88f2d3498f2a46ea3607d7 - start lsh shortlist

- [hihoan] 64707fa484c79fe68fdf5022b269b99d70bee5fd - Revert "start lsh shortlist"

- [github] ffd997e36015e1022a122095ebb8e147c5b59650 - Properly copy the entire vector in the int16_t case (#845)

- [Marcin.JunczysDowmunt] 7d1f941242928c976640a20f37e1bd9ac10011e8 - Merged PR 18309: Cleaner suppression of unwanted output words

- [Roman Grundkiewicz] c29cc83dc4d234f0e3a00a46a729053132b408b8 - Update submodule examples

- [Marcin.JunczysDowmunt] 4408e88a941b60e23ddb12e3afcf599ec0f42e9b - Merged PR 18366: Fix generation of special control characters for default vocabulary

- [Marcin Junczys-Dowmunt] bfa6180033307f579f91d4220971474d0c3de86d - Revert "remove TC_MALLOC from optional dependencies (#840)"

- [github] 0223ce90b1d6afaa047f2baeb4d0689f87d7ae81 - Fix Ubuntu GitHub checks (#848)

- [github] fddd0e0661eb13b2a132e401e268315a35e468f7 - Adds better Affine support for GPUs when using CUDA 11. Introduces a new bias addition kernel for CUDA < 11 (#778)

- [github] a17ee300f4567cb747a93b0559d48b664d0e882e - Create VERSION

- [github] be6506562321be49f8ef4b8c2a469662bbc1cfa4 - Allow to choose fine-grained CPU intrinsics on as CMake options (#849)

- [github] fdf9fe7d4a98268b5d451bc5b5944da959fbe8e2 - Update VERSION

- [Marcin.JunczysDowmunt] a05124176d8869962b717a3557c383406f8c76f4 - Merged PR 18531: Install GCC in Azure pipelines

- [Marcin.JunczysDowmunt] caddad90cdb06283da2f5d17a3340ca8c6387b38 - Merged PR 18505: RMSNorm on GPU

- [github] a7c3a0b2ef0d3a9ffd952bb1eb97e680633d1515 - fix beam_search ABORT when enable openmp and OMP_NUM_THREADS > 1 (#767)

- [Marcin Junczys-Dowmunt] 8a53b761d5bc922e4ab058a4487ad362d2edefaf - update version

- [github] bb6092da2b89c3882aa256480cef13081fd6f50f - Compute tensor size using integers (#851)

[View full output](http://vali.inf.ed.ac.uk/jenkins/job/marian-dev-cuda-10.2/78/)

**Comment by marianminion** [comment]
Build was fixed!
